<ui version="4.0" >
 <class>dlgPointDetails</class>
 <widget class="QDialog" name="dlgPointDetails" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>339</width>
    <height>226</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>Point Details</string>
  </property>
  <layout class="QVBoxLayout" >
   <property name="margin" >
    <number>9</number>
   </property>
   <property name="spacing" >
    <number>6</number>
   </property>
   <item>
    <layout class="QGridLayout" >
     <property name="margin" >
      <number>0</number>
     </property>
     <property name="spacing" >
      <number>6</number>
     </property>
     <item row="3" column="2" >
      <widget class="QLineEdit" name="txtZ" >
       <property name="maximumSize" >
        <size>
         <width>16777215</width>
         <height>16777215</height>
        </size>
       </property>
      </widget>
     </item>
     <item row="2" column="2" >
      <widget class="QLineEdit" name="txtY" >
       <property name="maximumSize" >
        <size>
         <width>16777215</width>
         <height>16777215</height>
        </size>
       </property>
      </widget>
     </item>
     <item row="1" column="2" >
      <widget class="QLineEdit" name="txtX" >
       <property name="maximumSize" >
        <size>
         <width>16777215</width>
         <height>16777215</height>
        </size>
       </property>
      </widget>
     </item>
     <item row="4" column="2" >
      <widget class="QLineEdit" name="txtW" >
       <property name="maximumSize" >
        <size>
         <width>16777215</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="text" >
        <string>1.0</string>
       </property>
      </widget>
     </item>
     <item row="0" column="2" >
      <widget class="QLineEdit" name="txtName" />
     </item>
     <item row="4" column="0" colspan="2" >
      <widget class="QLabel" name="label_4" >
       <property name="sizePolicy" >
        <sizepolicy>
         <hsizetype>5</hsizetype>
         <vsizetype>5</vsizetype>
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize" >
        <size>
         <width>60</width>
         <height>0</height>
        </size>
       </property>
       <property name="text" >
        <string>Weight</string>
       </property>
       <property name="margin" >
        <number>0</number>
       </property>
      </widget>
     </item>
     <item row="3" column="0" colspan="2" >
      <widget class="QLabel" name="label_3" >
       <property name="text" >
        <string>Z</string>
       </property>
      </widget>
     </item>
     <item row="2" column="0" colspan="2" >
      <widget class="QLabel" name="label_2" >
       <property name="text" >
        <string>Y</string>
       </property>
      </widget>
     </item>
     <item row="1" column="0" colspan="2" >
      <widget class="QLabel" name="label" >
       <property name="text" >
        <string>X</string>
       </property>
      </widget>
     </item>
     <item row="0" column="0" colspan="2" >
      <widget class="QLabel" name="label_5" >
       <property name="sizePolicy" >
        <sizepolicy>
         <hsizetype>5</hsizetype>
         <vsizetype>5</vsizetype>
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text" >
        <string>Name</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <spacer>
     <property name="orientation" >
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" >
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="dlgBtnBox" >
     <property name="orientation" >
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons" >
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::NoButton|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>txtName</tabstop>
  <tabstop>txtX</tabstop>
  <tabstop>txtY</tabstop>
  <tabstop>txtZ</tabstop>
  <tabstop>txtW</tabstop>
  <tabstop>dlgBtnBox</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>dlgBtnBox</sender>
   <signal>rejected()</signal>
   <receiver>dlgPointDetails</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel" >
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
